### DISCUSSION TITLE
How to change the rounded corner?

### DISCUSSION THREAD

**Discussion by nightire** [glideapps/glide-data-grid#570] [Q&A] [answered]
Currently, the data grid has rounded corners:

<img width="950" alt="SCR-20221207-su1" src="https://user-images.githubusercontent.com/1104018/206183265-3615efbc-55ab-4efd-807d-0a3dbc6aa523.png">

Is there's way to change the size? Or even get rid of them? I tried to figure it out in API docs and stories, but I don't think there is one; any help would be appreciated.

**Comment by nightire** [comment, marked as answer]
These rounded corners are only specified in stories. It's my fault, and I should try it before I ask.

**Reply by jassmith** [reply]
🖖
